terraform/environments/staging/relays.tf

```diff
@@ -1,157 +1,384 @@
-module "relays" {
-  count = var.relay_token != null ? 1 : 0
+locals {
+  subnet_ip_cidr_ranges = {
+    "africa-south1"           = "10.240.0.0/24",
+    "asia-east1"              = "10.240.1.0/24",
+    "asia-east2"              = "10.240.2.0/24",
+    "asia-northeast1"         = "10.240.3.0/24",
+    "asia-northeast2"         = "10.240.4.0/24",
+    "asia-northeast3"         = "10.240.5.0/24",
+    "asia-south1"             = "10.240.6.0/24",
+    "asia-south2"             = "10.240.7.0/24",
+    "asia-southeast1"         = "10.240.8.0/24",
+    "asia-southeast2"         = "10.240.9.0/24",
+    "australia-southeast1"    = "10.240.10.0/24",
+    "australia-southeast2"    = "10.240.11.0/24",
+    "europe-central2"         = "10.240.12.0/24",
+    "europe-north1"           = "10.240.13.0/24",
+    "europe-southwest1"       = "10.240.14.0/24",
+    "europe-west1"            = "10.240.15.0/24",
+    "europe-west2"            = "10.240.16.0/24",
+    "europe-west3"            = "10.240.17.0/24",
+    "europe-west4"            = "10.240.18.0/24",
+    "europe-west6"            = "10.240.19.0/24",
+    "europe-west8"            = "10.240.20.0/24",
+    "europe-west9"            = "10.240.21.0/24",
+    "europe-west10"           = "10.240.22.0/24",
+    "europe-west12"           = "10.240.23.0/24",
+    "me-central1"             = "10.240.24.0/24",
+    "me-west1"                = "10.240.25.0/24",
+    "northamerica-northeast1" = "10.240.26.0/24",
+    "northamerica-northeast2" = "10.240.27.0/24",
+    "northamerica-south1"     = "10.240.28.0/24",
+    "southamerica-east1"      = "10.240.29.0/24",
+    "southamerica-west1"      = "10.240.30.0/24",
+    "us-central1"             = "10.240.31.0/24",
+    "us-east1"                = "10.240.32.0/24",
+    "us-east4"                = "10.240.33.0/24",
+    "us-east5"                = "10.240.34.0/24",
+    "us-south1"               = "10.240.35.0/24",
+    "us-west1"                = "10.240.36.0/24",
+    "us-west2"                = "10.240.37.0/24",
+    "us-west3"                = "10.240.38.0/24",
+    "us-west4"                = "10.240.39.0/24"
+  }
+}
+
+# Create networks
+resource "google_compute_network" "network" {
+  project = module.google-cloud-project.project.project_id
+  name    = "relays-network"
+
+  routing_mode = "GLOBAL"
+
+  auto_create_subnetworks = false
+
+  depends_on = [
+    google_project_service.compute
+  ]
+}
+
+resource "google_compute_subnetwork" "subnetwork" {
+  for_each = local.subnet_ip_cidr_ranges
+  project  = module.google-cloud-project.project.project_id
+  name     = "relays-${each.key}"
```

Comment:
These names are unchanged, however, they shouldn't need to be unique globally, just per VPC network, so we should be OK.

Reply:
Turns out I was wrong about this. subnet names need to be globally unique too 🤦.